static/js/virtual_classroom.js

```
@@ -0,0 +1,1505 @@
+                currentInteractive = seat.id;
+                break;
+            }
+        }
+
+        // Update interaction state and popup
+        interactionState.nearInteractive = nearInteractive;
+        interactionState.currentInteractive = currentInteractive;
+
+        if (nearInteractive) {
+            let message;
+            if (currentInteractive && currentInteractive.startsWith('seat-')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)